packages/nodes-base/nodes/KoboToolbox/descriptions/Operations.ts

```diff
@@ -0,0 +1,130 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const operations = [
+			{
+				displayName: 'Operation',
+				name: 'operation',
+				type: 'options',
+				displayOptions: {
+					show: {
+						resource: [
+							'form',
+						],
+					},
+				},
+				options: [
+					{
+						name: 'Get',
+						value: 'get',
+						description: 'Retrieve a form definition',
+					},
+				],
+				default: 'get',
+				description: 'The operation to perform.',
+			},
+			{
+				displayName: 'Operation',
+				name: 'operation',
+				type: 'options',
+				displayOptions: {
+					show: {
+						resource: [
+							'submission',
+						],
+					},
+				},
+				options: [
+					{
+						name: 'Query',
+						value: 'query',
+						description: 'Query matching submissions',
+					},
+					{
+						name: 'Get',
```

Comment:
Rename collection to options

![image](https://user-images.githubusercontent.com/16496553/149586084-d16fd762-c78f-42cf-bcda-dc6a64d69eff.png)